chapters/develop-organize/guides/reading/README.md

```diff
@@ -5,31 +5,56 @@ Reviewer: Teo Duțu
 
 Guides are pieces of content consisting of detailed steps that are commonly used in practice.
 
-Guides are to be used by learners as tutorials, part of the learning process.
-They can be used as tutorial during practical session, with assistance provided to further explaining certain items, or as unassisted self-study.
+Guides are to be used by learners as **tutorials**, part of the learning process.
+They can be used as tutorials during ["Practical Sessions"](../../../use-deliver/practical-sessions/reading/README.md), either as unassisted self-study, or with assistance provided for better understanding.
 
-Guides can also by educators when demonstrating a topic or showcasing a real-world scenario to a live audience.
+Guides can also by educators as **demos**, when demonstrating a topic or showcasing a real-world scenario to a live audience.
 These demos are typically good ways to capture the attention of learners due to their real-world nature.
 
 Guides consist of:
 
-- text: created by following the guidelines in [the "Reading" section](../../reading/reading/README.md)
-- media: created by following the guidelines in [the "Media" section](../../media/reading/README.md)
-- support files: source code, skeleton files, application support files, fill-in-the blanks, templates
-
-A learner would read the text, view media files and use (hands-on) practice files to get replicate a practical result present by the guide.
+- description and walkthrough:
+  This are the rationale & goals of the guide (how it would help the learner) and the detailed set of steps that the learner must undertake to reach the goal.
```

Comment:
```suggestion
  These are the rationale & goals of the guide (how it would help the learner) and the detailed set of steps that the learner must undertake to reach the goal.
```